0.2-Elliptic-Curve-Math.ipynb

```diff
@@ -108,15 +108,11 @@
     "print(\"a + b = {}\\n\".format(ab))\n",
     "\n",
     "# ECKey() operations\n",
-    "# convert our ints to 32 byte representations since that is what ECKey expects\n",
-    "a_bytes = a.to_bytes(32,'big')\n",
-    "b_bytes = b.to_bytes(32,'big')\n",
-    "\n",
     "# use our bytes to instantiate ECKey instances \n",
     "a_key = ECKey() \n",
-    "a_key.set(a_bytes, True) # set the secret value of the Eckey\n",
+    "a_key.set(a, True) # set the secret value of the Eckey\n",
```

Comment:
Getting `TypeError: object of type 'int' has no len()` running through this. Im on Taproot_V0.1.2.

Reply:
Yeah it requires https://github.com/bitcoinops/bitcoin/pull/11